common/djangoapps/student/views.py

```diff
@@ -662,8 +662,16 @@ def dashboard(request):
             user, course_org_filter, org_filter_out_set
         )
 
+    if 'notlive' in request.GET:
+        redirect_message = _("The course you are looking for does not start until {0}.").format(
```

Comment:
@peter-fogg This doesn't conform to our i18n standards. Can you please review this and any other internationalized text you've merged, fix, and submit a PR? Feel free to tag me. Please see http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#use-named-placeholders (you should read the whole document, as developers are expected to understand it all).

@cahrens or @andy-armstrong if you want me to do a re-review of our i18n standards with your team, I'm happy to.